src/content/tutorials/fleet-management/job-management/_index.md

```diff
@@ -0,0 +1,487 @@
+---
+linkTitle: Job management
+title: Job management with Kueue
+description: Learn how to use Kueue for Kubernetes-native job queueing and resource management in Giant Swarm workload clusters to efficiently manage batch workloads, ML training jobs, and resource quotas.
+weight: 50
+menu:
+  principal:
+    parent: tutorials-fleet-management
+    identifier: tutorials-fleet-management-job-management
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+user_questions:
+  - How do I set up job queueing in my Giant Swarm cluster?
+  - What is Kueue and how does it help with resource management?
+  - How do I manage batch workloads and ML training jobs efficiently?
+  - How do I set up gang scheduling for distributed workloads?
+  - How do I configure all-or-nothing scheduling for coordinated jobs?
+  - How do I install and configure Kueue in Giant Swarm?
+last_review_date: 2025-10-16
+---
+
+Kueue is a Kubernetes-native system that manages quotas and how jobs consume them. It provides advanced job queueing, resource management, and fair sharing capabilities for batch workloads, machine learning training jobs, and other compute-intensive tasks. Giant Swarm supports Kueue through a managed app that simplifies installation and configuration.
+
+This guide explains how to set up and use Kueue for job management in Giant Swarm workload clusters.
+
+## Overview
+
+Kueue addresses the challenges of managing compute resources in multi-tenant Kubernetes environments by providing:
+
+### Why use Kueue
```

Comment:
it can also schedule/offload jobs into other clusters, right? maybe we can add that use case